### PR TITLE
Add : Reset date button

### LINKED ISSUE #80: Reset Button
### Description

- There must be a reset button to quickly reset the date 
- There are 2 places which look suitable for the reset button to me (any new suggestions are welcome).
- You must add the reset button in any one position, **remember to maintain the responsiveness of the page**.
- The function of this reset button will be to reset the dates to current date.
- The functionality must not be added right away only the button must be added in.

### Screenshots

Screenshot 2022-09-06 213724

### Additional information

- Please maintain the mobile responsiveness.
- Only use material UI components.
- Do not install any new Package.
- Make changes to the required file only.

## Happy contributing. 💐
Star my other Repositories [here](https://github.com/warmachine028)

### PR DESCRIPTION
**Pull request by amanchoudhary18**
# Issue  📐

- **I amanchoudhary18 have worked for #80**

[put x to check the boxes]: <> (This is a comment, it will not be included)

## Guidelines 🔐

**I accept the fact that i have followed the guidelines and have not copied the codes from around the internet**

- [x] **Contribution Guidelines**
- [x] **Code of Conduct**

## Issue to be closed 🛅

- **My pull request closes #80**

## Screenshots 📷

**Here are the pictures of changes that I have made 🔽**

<attach screenshots here>
  
image

  
  
---

### CODE REVIEW

File: client/src/components/studentModules/Bills/styles.js [hunk 1]
```diff
@@ -1,62 +1,70 @@
...
+    justifyContent: "space-between",
+    margin: "10px 0",
+    alignItems: "center",
+  },
+  [`& .${classes.resetDate}`]: {
+    display: "flex",
+    justifyContent: "space-between",
+    margin: "25px auto",
+    alignItems: "center",
+    backgroundColor: "#383562",
```
Comment (warmachine028) [inline client/src/components/studentModules/Bills/styles.js:29]: Can you leave the button color as Primary only, no need to add BackgroundColor property. Just remove it

**Review by warmachine028 — CHANGES_REQUESTED** [review]

File: client/src/components/studentModules/Bills/styles.js (hunk 1 above)
Comment (amanchoudhary18) [reply client/src/components/studentModules/Bills/styles.js:29]: Okay

Comment (amanchoudhary18) [reply client/src/components/studentModules/Bills/styles.js]: Done.

**Review by warmachine028 — APPROVED** [review]
Looks Good to me

**Comment by warmachine028** [comment]
Dont forget to star this repo 💚

**Pull request merged**